build.gradle

```diff
@@ -17,17 +17,19 @@ allprojects {
   apply plugin: 'groovy'
 
   spinnaker {
-    dependenciesVersion = "0.42.0"
+    dependenciesVersion = "0.45.0"
   }
 
   configurations.all {
     exclude group: 'javax.servlet', module: 'servlet-api'
     exclude group: 'javax.servlet', module: 'javax.servlet-api'
     resolutionStrategy {
-      force 'org.codehaus.groovy:groovy-all:2.4.5'
-      force 'org.spockframework:spock-core:1.0-groovy-2.4'
+      force 'org.codehaus.groovy:groovy-all:2.4.7'
       eachDependency {
-        if (it.requested.group == 'org.springframework') {
+        if (it.requested.group == 'org.spockframework') {
```

Comment:
was getting conflicts due to spinnaker-dependencies updating all of spock to 1.1 and our force only addressing spock-core